src/components/common/WithoutStylesMarkdownContent.test.tsx

```diff
@@ -39,9 +39,9 @@ describe('WithoutStylesMarkdownContent', () => {
         </Provider>
       </Router>,
     );
-    cy.get('div#__cy_root > div').children().should('have.length', 1);
-    cy.get('div#__cy_root > div > p').should('contain.text', 'A bad day');
-    cy.get('div#__cy_root > div > p').children().should('have.length', 0);
+    cy.root().should('have.length', 1);
+    cy.root().get('p').should('contain.text', 'A bad day');
```

Comment:
Just as a note, do you think this sort of test could maybe be simplified by doing a findByText on the page? That is what Terra-UI does

Reply:
Ah oops I missed this comment! It turns out `findByText` requires another dependency so I'm inclined to leave it out for now if that's okay